octobot_trading/personal_data/positions/position.py

```diff
@@ -260,6 +260,9 @@ def update_from_order(self, order):
         :param order: the filled order instance
         :return: the updated quantity, True if the order increased position size
         """
+        # consider the order filled price as the current reference price for pnl computation
+        self._update_mark_price(order.filled_price)
```

Comment:
It could have been filled 1 or 2sec ago and the mark price could have been updated. I don't think this is a good idea

Reply:
Then we need another solution for this. The issue is that positions can be closed by order that are not on the mark price. In this case we need to use the closing order price to compute the realized pnl since it's the one that really matters there (specifically on position update from a filled order).